src/components/SecondaryPanes/Expressions.css

```diff
@@ -63,6 +63,11 @@
   position: relative;
 }
 
+.expression-content .tree {
+  overflow: hidden;
+  max-width: calc(100% - 36px);
+}
```

Comment:
For the watch expressions, the case is more complex. Consider:
1. If we wanted the ellipsis style on expression label [2], we had to modify the shared ObjectInspector component [1]. 
2. What to do with all kinds of values nested under watched expression?
3. Would it bring undesired side effect to others after we touched this shared node module?

Put all the concerns together, only to control the overflow style of ObjectInspector component from Expression.css probably is a reasonable trade-off.

[1] https://github.com/devtools-html/devtools-core/blob/e81f5fe2267539dec1083a97009bffd45aa7a096/packages/devtools-reps/src/object-inspector/index.js#L449

[2] 
![screen shot 2017-09-19 at 6 57 22 pm](https://user-images.githubusercontent.com/5627487/30590101-c5c60f8c-9d6f-11e7-8bf4-2d30028a63c7.png)